cmd/tailscale/main.go

```diff
@@ -280,9 +286,14 @@ func (a *App) runBackend() error {
 	if err != nil {
 		return err
 	}
-	a.logIDPublicAtomic.Store(b.logIDPublic)
+	a.logIDPublicAtomic = b.logIDPublic
```

Comment:
If you assign to this, it's no longer functioning as an atomic. If assignment to this needs to be atomic, you should use `a.logIDPublicAtomic.Store()`.

Reply:
Done.